diagnostic.md

```diff
@@ -32,23 +37,30 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
+Getter:
+$('p').html()
+$('h1').text()
 
+Setter:
+$('p').html('this is p')
+$('h1').text('hello')
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+The DOM is a browser's rendering of a document organized hierarchically using nodes stored in local memory.
```

Comment:
This is close but it is not really the rendering that the browser does. It is more of a representation for the developer to use that affects how the browser renders.